tests/test_ci_xcodebuild_noninteractive_helper.py

```
@@ -209,6 +209,79 @@ def main() -> int:
+
+        print("Test Suite 'Selected tests' passed at now", flush=True)
+        print("\\t Executed 1 test, with 0 failures (0 unexpected) in 0.001 seconds", flush=True)
+        print("\\u25c7 Test run started.", flush=True)
+        for index in range(8):
+            print(f"\\u25c7 Test swiftTesting{index}() started.", flush=True)
+            time.sleep(0.1)
+        print("\\u2714 Test run with 8 tests in 1 suite passed after 0.8 seconds.", flush=True)
+        print("swift-testing-phase-complete", flush=True)
+        """
+    )
+    mixed_phase_result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)